sdk/cognitiveservices/cognitiveservices-luis-authoring/README.md

```diff
@@ -15,38 +15,38 @@ npm install @azure/cognitiveservices-luis-authoring
 
 ### How to use
 
-#### nodejs - Authentication, client creation and listPhraseLists features as an example written in TypeScript.
+#### nodejs - Authentication, client creation and listApplicationVersionPatternFeatures features as an example written in TypeScript.
 
-##### Install @azure/ms-rest-nodeauth
+##### Install @azure/ms-rest-azure-js
 
 ```bash
-npm install @azure/ms-rest-nodeauth
+npm install @azure/ms-rest-azure-js
 ```
 
 ##### Sample code
 
 ```typescript
-import * as msRest from "@azure/ms-rest-js";
-import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
-import { LUISAuthoringClient, LUISAuthoringModels, LUISAuthoringMappers } from "@azure/cognitiveservices-luis-authoring";
-const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
-
-msRestNodeAuth.interactiveLogin().then((creds) => {
-  const client = new LUISAuthoringClient(creds, subscriptionId);
-  const appId = ec7b1657-199d-4d8a-bbb2-89a11a42e02a;
-  const versionId = "testversionId";
-  const skip = 1;
-  const take = 1;
-  client.features.listPhraseLists(appId, versionId, skip, take).then((result) => {
+import { CognitiveServicesCredentials } from "@azure/ms-rest-azure-js";
+import { LUISAuthoringClient } from "@azure/cognitiveservices-luis-authoring";
+
+
+let subscriptionId = "b15ebe3a1ec446a08f8021fe6f95f0f6";
```

Comment:
In your other PR for authoring, you use `let subscriptionId = "<luis-authoring-key>";`
Can we be consistent and use `let subscriptionId = "<luis-runtime-key>";` or whatever makes sense instead of the actual id?

Reply:
Both of runtime and authoring sdks use luis authoring key to authenticate using `@azure/ms-rest-azure-js` package. so It makes more sense to use `let subscriptionId = "<luis-authoring-key>";` in both READMEs.